feat(quic): support stateless resets (#4554)
paritytech/rust-libp2p: identity/CHANGELOG.md

## 0.2.4 - unreleased

- Implement `Keypair::secret`, that returns the secret key of the keypair if it has one.

>   See [PR 4554]

Reviewer: Stale.

Author: right, thanks

## 0.2.4 - unreleased

> - Implement `Keypair::derive_secret`, that derives a secret from an input domain if the inner key
>   pair supports it.
>   See [PR 4554]

Reviewer: ```suggestion
- Implement `Keypair::derive_secret`, to deterministically derive a new secret from the embedded secret key.
  See [PR 4554].
```
